java-generator/core/src/main/java/io/fabric8/java/generator/nodes/AbstractJSONSchema2Pojo.java

```diff
@@ -54,10 +60,31 @@ public String getDescription() {
     return description;
   }
 
-  protected AbstractJSONSchema2Pojo(Config config, String description, final boolean isNullable) {
+  protected String getClassType() {
+    return getType();
+  }
+
+  /**
+   * This method is responsible for creating a block statement that
+   * represents the body of a default value initialization method.
+   *
+   * @return a {@link com.github.javaparser.ast.stmt.BlockStmt} instance that contains all the statements that code the
+   *         default value initialization.
+   */
+  protected BlockStmt generateDefaultInitializerBody() {
```

Comment:
I understand that this request might sounds a little "odd", but I would prefer if we can keep this encoding in the "parent" class (e.g. probably is sufficient a `private` method in JObject) and expose the `defaultValue` using a Getter.

The reason is:
currently the recursion is leveraging a single code "brach" that is implemented using `generateJava`, keeping it uniform makes it easier to refactor in the future and improve performance on the hot path.

Do you see any problem in implementing it in this way?

Reply:
I do agree, fixed.